src/remoted/manager.c

```diff
@@ -106,8 +106,11 @@ void save_controlmsg(const keyentry * key, char *r_msg, size_t msg_length)
     char *config_sum = NULL;
     char *merged_sum = NULL;
     char *agent_ip = NULL;
-    char *labels = NULL;
     char manager_host[512] = "";
+    char *labels = NULL;
+    const char * AGENT_IP_LABEL = "#\"_agent_ip\":";
+    const char * MANAGER_LABEL = "#\"_manager_hostname\":";
+    const char * NODE_LABEL = "#\"_node_name\":";
```

Comment:
Style question: aren't the words with uppercase letters reserved for global definitions?

Reply:
Fixed